docs/_data/toc.yml

```diff
@@ -113,196 +119,199 @@
-          url: guides/creating-a-meshery-design
-        - title: Working with Designs
-          url: guides/configuration-management/working-with-designs
+          url: guides/configuration-management/creating-a-meshery-design
         - title: Importing Designs
-          url: guides/configuration-management/importing-designs
-        - title: Working with design patterns
+          url: guides/configuration-management/importing-models
+        - title: Exporting Designs
+          url: guides/configuration-management/exporting-models
+        - title: Generating Designs
+          url: guides/configuration-management/generating-models
+        - title: Managing Model Images
+          url: guides/configuration-management/push-pull-model-image
+        - title: Cloud Native Design Patterns
           url: guides/configuration-management/pattern-management
-    - title: Managing Events with Notification Center
-      url: guides/events-management
-    - title: GitOps with Meshery
-      url: guides/infrastructure-management/gitops-with-meshery
     - title: Performance Management ↆ
-      url: guides/performance-management
+      url: guides/performance-management/performance-management
       children:
-        - title: Performance Management with Meshery
-          url: guides/performance-management/performance-management
         - title: Using Metrics in Meshery
           url: guides/performance-management/meshery-metrics
+        - title: Managing Performance
+          url: guides/performance-management/managing-performance
         - title: Interpreting Performance Test Results
           url: guides/performance-management/interpreting-performance-test-results
-        - title: Performance Profiles and Tests #should go under concepts
-          url: guides/performance-management/managing-performance
     - title: Using Meshery CLI Guides ↆ
       url: guides/mesheryctl
       children:
         - title: Using mesheryctl
           url: guides/mesheryctl/working-with-mesheryctl
-        - title: Configure autocompletion for mesheryctl
+        - title: Autocompletion Setup
           url: guides/mesheryctl/configuring-autocompletion-for-mesheryctl
-        - title: Authenticate via Meshery CLI
+        - title: CLI Authentication
           url: guides/mesheryctl/authenticate-with-meshery-via-cli
         - title: Running system checks using Meshery CLI
           url: guides/mesheryctl/running-system-checks-using-mesheryctl
+        - title: System Commands
+          url: guides/mesheryctl/system-commands
     - title: Troubleshooting Guides ↆ
       url: guides/troubleshooting
       children:
-        - title: Installation
+        - title: Installation Issues
           url: guides/troubleshooting/installation
-        - title: Meshery Operator
+        - title: Operator & MeshSync
           url: guides/troubleshooting/meshery-operator-meshsync
-        - title: Meshery Server
+        - title: Runtime Errors
```

Comment:
Does "Runtime" include or exclude Meshery Operator and MeshSync?

As a reader, browsing the Troubleshooting section, I can't say that I know what topic to expect being "Runtime Errors", while I do know what topic to expect behind "Meshery Server".